### PR TITLE
Feature/eng 1176 domain tests

### PR DESCRIPTION
**Pull request by nilsbeck**
# Description
This PR adds `model.MaxInt` and `model.MinInt` to be used as platform independent integers. These are needed for new tests in plugins which will be added in [this PR](https://github.com/nextmv-io/plugins/pull/61)

## Changes
- adds `model/const.go`

### CODE REVIEW

**Review by larsbeck — DISMISSED** [review]

File: model/const.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// related proprietary information is strictly prohibited without the express
+// written permission of nextmv.io, inc.
+
+package model
+
+import "math/bits"
+
+// Constants for integer bounds.
+const (
+	MaxInt int = (1<<bits.UintSize)/2 - 1
```
Comment (sebastian-quintero) [inline model/const.go]:
```suggestion
	// MaxInt is the maximum value for an integer.
	MaxInt int = (1<<bits.UintSize)/2 - 1
```
Add godoc here plz.

File: model/const.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	MinInt     = (1 << bits.UintSize) / -2
```
Comment (sebastian-quintero) [inline model/const.go]:
```suggestion
	// MinInt is the minimum value for an integer.
	MinInt     = (1 << bits.UintSize) / -2
```
Same here bitte.

**Review by sebastian-quintero — CHANGES_REQUESTED** [review]
I don't think that a new file is worthwhile. You can throw these at the top of the existing `model.go` file, up to you.

**Review by larsbeck — DISMISSED** [review]

File: model/model.go [hunk 3]
```diff
@@ -1,5 +1,15 @@
 package model
 
+import "math/bits"
+
+// Constants for integer bounds.
+const (
+	// MaxInt is the maximum value for an integer.
+	MaxInt int = (1<<bits.UintSize)/2 - 1
+	// MinInt is the minimum value for an integer.
+	MinInt = (1 << bits.UintSize) / -2
```
Comment (merschformann) [inline model/model.go:10]: Funny how the formatter thinks one of the two is long enough to compress `<<` and for the other it's not. :joy_cat:

**Review by merschformann — DISMISSED** [review]
:rocket:

**Review by sebastian-quintero — DISMISSED** [review]

**Review by sebastian-quintero — APPROVED** [review]

**Review by merschformann — APPROVED** [review]

**Pull request merged**